### ISSUE TITLE
Flask Website Iteration

### ISSUE THREAD

**Issue by holantonela** [MetaMask/metamask-extension#13562]

**Comment by Montoya** [comment]
This CTA is confusing (screenshot taking on Chrome for Desktop) 
![image](https://user-images.githubusercontent.com/19909/153051737-f35705bd-a795-4cbe-bd13-e5be62ed2dfd.png)

**Comment by holantonela** [comment]
What do you expect to have there @Montoya? a `Download Now`? What would you suggest for non Chrome Desktop environments?

**Comment by Montoya** [comment]
@holantonela yes, on Chrome Desktop I would expect to see "Download now" and on other platform I would expect the message that is shown in the screenshot

**Comment by Montoya** [comment]
This is done! Changes were made to /flask and a new webpage was launched /snaps

A list of selected available snaps may be included in a future update
